server/app.js

```diff
@@ -12,6 +14,12 @@ const ReportUtils = require('./utilities/ReportUtils.js');
 http.listen(3000, () => {
     let users = DataAccessLayer.ReadUsersFile();
     let reports = DataAccessLayer.ReadReportsFile();
+
+    let numberOfEachRoom = 2;
```

Comment:
What is this variable supposed to represent? The name is slightly confusing

Reply:
I've added a comment in a version I just pushed